app/components/icon-catalog-form.ts

```diff
@@ -17,10 +17,15 @@ export default class IconCatalogFormComponent extends ImageUploadHandlerComponen
 
   @action
   async setIconValue(attributeName: keyof IconModel, event: InputEvent) {
-    await this.args.icon.set(
-      attributeName,
-      (event.target as HTMLInputElement).value,
-    );
+    // remove leading spaces so they don't mess up the sorting mechanism
+    const inputElement = event.target as HTMLInputElement;
+    const trimmedValue = inputElement.value.trimStart();
+    if (inputElement.value !== trimmedValue) {
+      inputElement.value = trimmedValue;
+    }
```

Comment:
Is this needed? I would expect that the value is also updated by Ember, if it changed.

Reply:
I don't see how i can fix this issue. Even tried saving the icon model before setting the value, but it still allows me to save untrimmed input. Should we just leave it as is ?

Reply:
Can you push the latest code (I think not all changes are pushed at least), I'll try to take a look.

Reply:
Update, the issue was caused by the 2-way-binding feature of the Appuniversum (v2) AuInput component. The version/2.0.0 branch already resolved this issue with the Appuniversum v3 update, so we can just update this branch as well.